coordinator/src/db/user.rs

```diff
@@ -49,26 +51,59 @@ pub fn by_id(conn: &mut PgConnection, id: String) -> QueryResult<Option<User>> {
 pub fn upsert_user(
     conn: &mut PgConnection,
     trader_id: PublicKey,
-    contact: String,
+    contact: Option<String>,
+    nickname: Option<String>,
 ) -> QueryResult<User> {
+    // If no name or contact has been provided we default to empty string
+    let contact = contact.unwrap_or_default();
+    let nickname = nickname.unwrap_or_default();
```

Comment:
🤔 What is the advantage of this?

I think it would be better to model this as a nullable type, so that it's more explicit when the user has decided to not provide one of these.